packages/workbox-sw/test/sw/router.js

```diff
@@ -157,4 +157,19 @@ describe(`Test workboxSW.router`, function() {
     const route = workboxSW.router.registerRoute(() => {}, () => {});
     expect(route).to.exist;
   });
+
+  it(`should support calling get()`, function() {
+    const workboxSW = new WorkboxSW();
+    return new Promise((resolve) => {
+      const route = workboxSW.get('/', (args) => resolve());
```

Comment:
Calling resolve here will return the test as a success before do the expect checks. You'll probably want to add the route checks to `new Promise().then(() => { /*checks here */});`

Reply:
Even if I'm only doing static checks on the `route` object itself (see below)?

I adapted [this previous test](https://github.com/GoogleChrome/workbox/blob/master/packages/workbox-sw/test/sw/router.js#L80) from the same file.

Reply:
You are right, although if that's all you are doing, then you don't need the outer promise at all or the FetchEvent etc.